src/components/pages/service/support.tsx

```diff
@@ -69,5 +65,19 @@ export const ExpandableStyled = styled(Expandable)`
 export const UnorderedList = styled.ul`
   line-height: 150%;
   margin: 0;
-  padding-left: 1em;
+  padding-left: 0;
+  list-style-type: none;
+`;
+
+export const DashedListItem = styled.li`
```

Comment:
Not totally sure about this but: All `li` children's of the unordered list should always have the `-` right? I think it makes sense to put the styled then inside the `UnorderdList` component. 

Should look something like this (not sure about the correct syntax, but you will figure it our ;-)):
```
export const UnorderedList = styled.ul`
  ...
  li > {
    ... your styles
  }
`;
```

This way, we don't have to use this custom component for the list items and we are sure that the style is used on all our unurderd lists.